app/Commands/DefaultCommand.php

```diff
@@ -43,6 +43,7 @@ protected function configure()
                     new InputOption('dirty', '', InputOption::VALUE_NONE, 'Only fix files that have uncommitted changes'),
                     new InputOption('format', '', InputOption::VALUE_REQUIRED, 'The output format that should be used'),
                     new InputOption('cache-file', '', InputArgument::OPTIONAL, 'The path to the cache file'),
+                    new InputOption('with-exit-status', '', InputOption::VALUE_NONE, 'Only fix files that have uncommitted changes'),
```

Comment:
I think the description here might be incorrect (copied and pasted from a few lines above?)

Reply:
Oops, didn't even see that. I'll fix that up and write any relevant tests as there seems to be interest in this feature.